doc/continuous_deployment/Devenv_in_local_env.md

```diff
@@ -0,0 +1,57 @@
+# Devenv in local environment
+
+This document describes the steps required to provision a development environment in local VMware environment.
+
+## Testbed setup
+
+1. Clone 2 VMs from `win2016core-bare` template
+    - single ESXi host
+    - destination folder: `Dev\[initials]`
+    - datastore: `esxi*-main`
+    - management network: `v1800_VM`
```

Comment:
In "Select clone options" tab, choose "Customize this virtual machine's hardware".
In "Customize hardware" tab, set 
* "Network Adapter 1" to "v1800_VM" (this will be management network)
* "Network Adapter 2" to a network name like `vS-DevEnvX-Y`. If no such network is on the list, click Show more networks. In the network name
- `X` corresponds to ESXi host index chosen in "Select Compute resource" tab. 
- `Y` is a network number

Reply:
"review" is WIP, but going for lunch.